src/Components/App/App.css

```diff
@@ -1,9 +1,37 @@
 :root {
-  --primary-bkgd-color: rgb(255, 204, 0);
-  --secondary-bkgd-color: rgb(254, 84, 84);
-  --tertiary-bkgd-color: rgb(230, 229, 229);
-  --hover-bkgd-color: ivory;
-  --bkgd-radial-gradient: content-box radial-gradient(var(--secondary-bkgd-color), var(--primary-bkgd-color));
+  --element-bkgd-color: rgba(255,255,255, .4);
+  --border-radius: 2px;
+  --font-family: 'Plus Jakarta Sans', sans-serif;
```

Comment:
Love love love this

Reply:
Huge fan of the css variables! Great organization!